### ISSUE TITLE
Documentation for tf.nn.ctc_* `label` argument is unclear

### ISSUE THREAD

**Issue by sstirlin** [tensorflow/tensorflow#9521]
The documentation for the connectionist temporal classifiers is unclear for `label` argument.  Here is what exists currently:

```
labels: An int32 SparseTensor. labels.indices[i, :] == [b, t] means labels.values[i] stores the id for (batch b, time t). labels.values[i] must take on values in [0, num_labels). See core/ops/ctc_ops.cc for more details.
```

The only way I was able to figure it out is from Jerod's comment on this SO:

http://stackoverflow.com/questions/42488070/how-to-design-the-label-for-tensorflows-ctc-loss-layer

**Comment by aselle** [comment]
Thanks for raising this issue. Do you have any suggestions on how to make it clearer? Perhaps you could provide a PR to make it clearer. If you know what the SparseTensor is, even having even this description is redundant (it seems not scalable to include detailed documentation on the semantics of sparse tensors in every routine that takes one). Should it perhaps link to the SparseTensor documentation?

**Comment by girving** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by selcouthlyBlue** [comment]
I suppose it would be nice to give an example on how to prepare a sequence of labels (e.g. characters of a sentence) to be encoded using some character set, and converted into a `SparseTensor` to be passed into the ctc loss function.
